xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-api/src/main/java/org/xwiki/search/solr/AbstractSolrCoreInitializer.java

```diff
@@ -138,25 +143,34 @@ public abstract class AbstractSolrCoreInitializer implements SolrCoreInitializer
 
     private static final String SOLR_VERSIONFIELDTYPE_VALUE = "defVal";
 
+    private static final int MIGRATION_BATCH_ROWS = 100;
+
     @Inject
     protected ComponentDescriptor<SolrCoreInitializer> descriptor;
 
     @Inject
-    protected SolrSchemaUtils solrSchemaUtils;
+    protected SolrUtils solrUtils;
 
-    protected SolrClient client;
+    protected XWikiSolrCore core;
 
-    protected Map<String, FieldTypeRepresentation> types;
+    @Deprecated(since = "16.1.0RC1")
```

Comment:
Missing some doc to explain what to use now it's deprecated: I saw in one of the initializer that the call should now be `this.core.getClient()` right?